.github/workflows/push_to_gcloud.yml

```diff
@@ -24,12 +24,20 @@ jobs:
     - name: Install python packages
       run: |
         pip install jinja2==2.11.3 PyYAML==5.4.1 jsonschema==3.2.0 pandas==1.1.4
+    # Create the secrets.csv from the contents of the `AGENCY_SECRETS` environment variable / GitHub
```

Comment:
Can we separate this into its own PR? AFAIK the API key spreadsheet is private (done under the rationale that the API keys are read keys, so need to be inaccessible outside the org, but can be widely accessible inside, if it makes filling the spreadsheet out)

This is a big enough change that it probably should be done via a quick sync + an issue

Reply:
I have removed this logic and will create another PR for this.